src/Upgrader/Business/UpgraderBusinessFactory.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * Copyright © 2021-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrader\Business;
+
+use Upgrader\Business\Composer\ComposerClient;
+use Upgrader\Business\Git\GitClient;
+use Upgrader\UpgraderConfig;
+
+class UpgraderBusinessFactory
```

Comment:
Can we have one factory that is responsible for objects initialisation?

Reply:
We have one factory on the Business layer and one factory for each client (ComposerClient, GitClinet, etc....)